app.js

```diff
@@ -3,11 +3,21 @@ const app = express();
 const {
     getTopics
 } = require('./db/controller/topic-controller');
+const {
+    getArticles
+} = require('./db/controller/article-controller');
 const { getEndpointInformation } = require('./db/controller/endpoints-controller');
 
 app.use(express.json());
```

Comment:
You only have GET requests at the moment. Should you have express.json in here yet?